tests/pyLDAvis/test_gensim_models.py

```diff
@@ -59,3 +59,4 @@ def test_hdp():
 if __name__ == "__main__":
     test_lda()
     test_hdp()
+    
```

Comment:
Please remove  those files from the commit if they don't add any new code. E.g. both the test files don't have any new code. Although, I would suggest you add some test code so that we can try out your implementation.